lib/puppetserver/ca/action/setup.rb

```diff
@@ -77,8 +77,8 @@ def generate_pki(settings, signing_digest)
           ca = Puppetserver::Ca::LocalCertificateAuthority.new(signing_digest, settings)
 
           root_key, root_cert, root_crl = ca.create_root_cert
```

Comment:
nit: this could be a class method

Reply:
I totally agree; I kinda thought that much of the root_* aspects of this class should be removed entirely and put into some kind of Util...